test/conformance/kuttl/verifyImages/clusterpolicy/standard/noconfigmap-diffimage-success/update-policy.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  annotations:
+    pod-policies.kyverno.io/autogen-controllers: none
+  name: image-verify-polset
+spec:
+  background: false
+  failurePolicy: Fail
+  rules:
+  - context:
+    - configMap:
+        name: myconfigmap1
+        namespace: mynamespace
+      name: myconfigmap1
+    match:
+      resources:
```

Comment:
Same comment about `any` or `all`

Reply:
fixed

Reply:
@shahpratikr - I don't see any change to add `any` or `all`, did you push the change?

Reply:
yes just now pushed it